src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -81,6 +83,13 @@ def load(self, path_lookup: PathLookup) -> SourceContainer | None:
     def __repr__(self):
         return f'WorkflowTask<{self._task.task_key} of {self._job.settings.name}>'
 
+    @property
+    def lineage(self) -> Lineage:
+        job_name = ("" if self._job.settings is None else self._job.settings.name) or ""
+        job_lineage = LineageAtom("job", str(self._job.job_id), {"name": job_name})
```

Comment:
```suggestion
        job_lineage = LineageAtom("job", str(self._job.job_id), name=job_name)
```
i thought that `**other` would allow that, no?...

Reply:
works with a method but not with the default dataclass constructor